test/utils/helpers.js

```diff
@@ -0,0 +1,14 @@
+const testWillThrow = async (contractFunction, input, args, errorMessage) => {
+	try {
+		await contractFunction(...input, args)
+
+		assert(false, 'contract failed to throw an error')
+	} catch (error) {
+		assert(
+			RegExp(errorMessage).test(error),
+			`the error message => ${errorMessage} => does not match => ${error}`
+		)
+	}
+}
+
+module.exports = { testWillThrow } 
```

Comment:
Minor, should have a newline. Could you add this requirement to the linter as well?

Reply:
Ace, good catch!

Added [eol-last](https://eslint.org/docs/rules/eol-last) to the eslintrc :+1: